app/views/templates/section_home.html.haml

```diff
@@ -2,3 +2,12 @@
   - breadcrumb :public_node, node
 
   = render partial: 'templates/content_body', object: node.content_body
+
+  - if @section.topics.any?
+    %h4 Popular now
+    %ul.list-vertical--thirds
+    - @section.topics.by_name.each do |topic|
```

Comment:
Do only agencies have topics? If so does this template work for all section home pages? Also I tried to test it and there's no way in admin to add a topic to an agency. It might be worth doing that as well.